config/example/node.env

```diff
@@ -1,10 +1,10 @@
 NEOFS_LOGGER_LEVEL=debug
 
 NEOFS_PROFILER_ADDRESS=127.0.0.1:6060
-NEOFS_PROFILER_SHUTDOWN_TIMEOUT=15s
+NEOFS_PROFILER_SHUTDOWN_TIMEOUT=30s
 
 NEOFS_METRICS_ADDRESS=127.0.0.1:9090
-NEOFS_METRICS_SHUTDOWN_TIMEOUT=15s
+NEOFS_METRICS_SHUTDOWN_TIMEOUT=30s
```

Comment:
What's wrong with 15s timeouts?

Reply:
set default value

Reply:
It's better to *avoid* default values in this config, but keep values sane in the same time.
Configuration test fetches some *non default* values and compare them correctly.
To check default values configuration test uses empty config.

Reply:
forget that there are tests for default values

rolled back and changed other default values